presto-parser/src/main/java/io/prestosql/sql/ExpressionFormatter.java

```diff
@@ -506,10 +506,8 @@ protected String visitLikePredicate(LikePredicate node, Void context)
                     .append(" LIKE ")
                     .append(process(node.getPattern(), context));
 
-            node.getEscape().ifPresent(escape -> {
-                builder.append(" ESCAPE ")
-                        .append(process(escape, context));
-            });
+            node.getEscape().ifPresent(escape -> builder.append(" ESCAPE ")
```

Comment:
It was a lambda already. Change `Use lambda expression` commit message to `Simplify lambda expression`.